src/components/VestingControl.tsx

```diff
@@ -48,11 +48,14 @@ const useStyles = makeStyles({
 export const VestingControl: React.FC<Props> = (props) => {
 	const context = useWeb3React();
 	const classes = useStyles();
-	const contracts = useContracts();
-	const vestingContract = contracts!.vesting;
+
+	//This lines must be changed so the user can set with ui 
+	//wich contract wants to send transactions, hardcoded actually to test
+	const contracts: Maybe<Contracts[]> = useContracts();
+	const vestingContract = contracts![0];
```

Comment:
Instead of hardcoding 0, use a React state variable, or have the number be passed in.